scout/artifactory.md

```diff
@@ -0,0 +1,171 @@
+---
+description: >
+  Integrate JFrog Artifactory and JFrog Container Registry with Docker Scout
+keywords: >
+  docker scout, jfrog, artifactory, jcr, integration, image analysis, security,
+  cves
+title: Artifactory integration
+---
+
+> **Note**
+>
+> Docker Scout is an [early access](../release-lifecycle.md#early-access-ea)
+> product, and requires a Pro, Team, or Business subscription.
```

Comment:
Here I think it might be simpler to say "...early access product, and requires a subscription" since user will need a core Docker subscription, plus a subscription to Scout according to their level of use,